scripts/prompt-config.sh

```diff
@@ -1358,8 +1394,15 @@ Channel to be used for software update. Options:
 		printf "NTP servers: $ntp_hosts\n"
 		echo
 		printf "Enable telemetry: $phonehome_automatic\n"
-		echo
 		printf "Update channel: $update_channel\n"
+		if [[ -n ${external_nic} ]]; then
```

Comment:
Aha, so you went with, "don't print anything if no external nic," right?

Reply:
Yeah.